crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -36,24 +36,68 @@ fn ambiguous_with(graph_info: &mut GraphInfo, set: InternedSystemSet) {
     }
 }
 
+/// Stores data to differentiate different Node types
+pub trait NodeType {
+    /// Used in `NodeConfig`
```

Comment:
```suggestion
    /// Additional data used to configure a node. Stored in [`NodeConfig`].
```

We could elaborate here a bit.

BTW, (for bevy at least) 99% of the time you want to wrap anything quoted in ticks with square brackets so rustdoc picks them up.